app/presenters/evidence_summary_presenter.rb

```diff
@@ -0,0 +1,35 @@
+class EvidenceSummaryPresenter < BasePresenter
+  def link
+    h.link_to title, full_path
```

Comment:
I know we spoke about this re the BasePresenter pr but I'm having second thoughts about using `h`. `view` might be better, particularly as we used `view` in wpcc. wdyt @willhall88 ?

Reply:
I am happy either way. @willhall88 wdyt?

Reply:
so I first thought `h` here referred to the alias for `html_escape` http://api.rubyonrails.org/classes/ERB/Util.html#method-c-h
even though that's a method and not an object.. 
I think `view` or similar would be better understood

Reply:
I renamed to view now. I leave the commit without squash because IMHO this commit of renaming to use view should be in the history.